src/test/scala/cognite/spark/v1/FlexibleDataModelCorePropertyRelationTest.scala

```diff
@@ -493,20 +825,36 @@ class FlexibleDataModelCorePropertyRelationTest
       cursor = ""
     )
 
-    val readAllDf = readRows(
```

Comment:
I don't quite understand how this was working before. Reading the code it looks like we are clearly requesting instances of viewAll but of type edge and node, and then combining them. This should return only instances of viewAll which there are 2, not 8, yet it's expected later to contain 8 elements?